### PR TITLE
Fix/all files rendering

### LINKED ISSUE #243: Generating a changelog with files in `.changelogs` wil fail unless they all contain the same/correct frontmatter
This is especially important now that we can have a `suffix.md` file.

### PR DESCRIPTION
**Pull request by TheNeikos**
This fixes #243

### CODE REVIEW

File: src/command/generate_changelog_command.rs [hunk 1]
```diff
@@ -85,16 +86,21 @@ fn load_release_files(
...
         .same_file_system(true)
         .into_iter()
-        .filter_map(|rde| match rde {
+        .filter_map(move |rde| match rde {
             Err(e) => Some(Err(e)),
             Ok(de) => {
                 if de.file_type().is_file() {
-                    if de.path().ends_with("template.md") || de.path().ends_with(".gitkeep") {
+                    // Since the file itself comes from the workdir, this cannot fail?
+                    let clean_path = de.path().strip_prefix(root_path.as_path()).unwrap();
```
Comment (matthiasbeyer) [inline src/command/generate_changelog_command.rs:100]: Unwrap not nice. :laughing:

File: src/command/generate_changelog_command.rs [hunk 2]
```diff
@@ -85,16 +86,21 @@ fn load_release_files(
...
+                    if clean_path.components().count() <= 1
```
Comment (matthiasbeyer) [inline src/command/generate_changelog_command.rs:101]: What exactly does this check?

**Review by matthiasbeyer — CHANGES_REQUESTED** [review]
I like the changes, only two minor things.

File: src/command/generate_changelog_command.rs (hunk 2 above)
Comment (TheNeikos) [reply src/command/generate_changelog_command.rs:101]: It checks that you are not in a subdirectory and returns None.

File: src/command/generate_changelog_command.rs (hunk 1 above)
Comment (TheNeikos) [reply src/command/generate_changelog_command.rs:100]: What do you suggest? This can't fail

Comment (matthiasbeyer) [reply src/command/generate_changelog_command.rs:100]: Well, that's true.

**Review by matthiasbeyer — APPROVED** [review]

**Comment by matthiasbeyer** [comment]
bors merge

**Pull request merged**